docs/changelog.rst

```diff
@@ -2,6 +2,8 @@
 Changelog
 =========
 
+* :bug:`4034` If a new premium user is created, with DB sync on, premium api keys
+being correct but wrong password is given then creating same user with right password will now work.
```

Comment:
The changelog script will fail since it expects a line per change

Reply:
Docs failing is EXACTLY because of this.

```
preparing documents... done
/home/lefteris/w/rotkehlchen/docs/changelog.rst:6: WARNING: unknown node type: <bug #4013>                                                                                                                                                                                       

Exception occurred:
  File "/home/lefteris/.virtualenvs/rotkipy37/lib/python3.7/site-packages/docutils/nodes.py", line 2058, in unknown_departure
    % (self.__class__, node.__class__.__name__))
NotImplementedError: <class 'sphinx.writers.html5.HTML5Translator'> departing unknown node type: Issue
The full traceback has been saved in /tmp/lefteris/sphinx-err-5t8i8qph.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
make: *** [Makefile:19: html] Error 2
```